templates/ignition-ec2.template.yaml

```diff
@@ -443,7 +443,7 @@ Resources:
       UserData: !Base64
         Fn::Sub: |
             #!/bin/bash
-            aws s3 cp s3://${QSS3BucketName}/${QSS3KeyPrefix}scripts/creation_v2.sh .
+            wget https://s3.amazonaws.com/${QSS3BucketName}/${QSS3KeyPrefix}scripts/creation_v2.sh
```

Comment:
@iatraviscox - this will not pass in our CI because the test buckets are private. Why do you need to switch from aws cli to wget?

Reply:
The aws cli didn't work. Failed everytime we tried to launch for some reason. You should try the current template.

Reply:
@iatraviscox - I just tested it, worked fine for me.  See output attached.
[taskcat_outputs-2023-06-06.zip](https://github.com/aws-quickstart/quickstart-inductive-automation-ignition/files/11667861/taskcat_outputs-2023-06-06.zip)

Reply:
I just tried it on my AWS account and it failed on the EC2 step. With the wget it worked.

Reply:
I tried the QuickStart on my AWS account too and seems like User Data script in EC2Ignition Instance is erroring out with this message: "Received FAILURE signal with UniqueId i-068ae4187c4fc5e03"